Resources/Prototypes/Reagents/narcotics.yml

```diff
@@ -300,7 +300,7 @@
         conditions:
         - !type:ReagentThreshold
           reagent: Nocturine
-          min: 8
+          min: 19
```

Comment:
Changing this to 11 will achieve your desired effect of making the user use 3 clicks without totally ruining Noct.

Reply:
> Changing this to 11 will achieve your desired effect of making the user use 3 clicks without totally ruining Noct.

good idea man